src/components/hashtags/Hashtags.js

```diff
@@ -10,12 +10,12 @@ import './hashtags.css';
 
 export default class Hashtags extends React.Component {
   renderData() {
-    const { hashtags, keyword, tweets_count_ts, onTagClick } = this.props;
+    const { hashtags, keyword, timePeriod, tweets_count_ts, onTagClick } = this.props;
 
     return(
       <React.Fragment>
         <div className='wordsWrapper'>
-          <Words hashtags={hashtags} keyword={keyword} onChange={onTagClick} />
+          <Words hashtags={hashtags} keyword={keyword} timePeriod={timePeriod} onChange={onTagClick} />
```

Comment:
no need to pass this, move the callback to a `Hashtags` class and pass it as `onChange`